docs/navigate/tools-diagnostics/toc.yml

```diff
@@ -1902,6 +1902,9 @@ items:
           - name: SYSLIB1230
             href: ../../fundamentals/syslib-diagnostics/syslib1230.md
             displayProperty: syslib1230, syslib1231, syslib1232, syslib1233, syslib1234, syslib1235, syslib1236, syslib1237, syslib1238, syslib1239
+          - name: SYSLIB5003
+            href: ../../fundamentals/syslib-diagnostics/syslib5003.md
+            displayProperty: syslib5003
```

Comment:
This should be moved up into the "Experimental features" section which is immediately above the "Source-generated code" section.